### ISSUE TITLE
Segregation of storage folder for RTB & CRTB on Azure, as it was there before migration to BC365

### ISSUE THREAD

**Issue by TabrejAlam3K** [3ktech/General#456]

**Comment by TabrejAlam3K** [comment]
Mr. Rajkumar to provide access to Mr. Kartik D - It is still pending by Client ( Rajkumar ) Side.

CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Hi Mr. Kartik D @3ktech - This task is on top priority.

CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Hi Mr. @3ktech (Kartik D) - Please update the progress of this task. Mr. Murali is working on.

**Comment by TabrejAlam3K** [comment]
Testing is in progress with the LCMS User ( Somendra).

**Comment by TabrejAlam3K** [comment]
Need to Deploy into the Production



CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Mr. Suvasis need to change the storage account folder path on the production environment, then it will working fine. we have already tested in UAT with the client.

CC : @audhi3k

**Comment by TabrejAlam3K** [comment]
I am closing this ticket because new git ticket has been created.

Related New Ticket is #544
